play/src/pusher/data/companions.json

```diff
@@ -1,31 +1,48 @@
-[
-    {
+{
+  "companion": {
+    "collections": [
+      {
         "name": "default",
+        "position": 0,
         "textures": [
-            {
-                "name": "dog1",
-                "img": "resources/characters/pipoya/Dog 01-1.png"
-            },
-            {
-                "name": "dog2",
-                "img": "resources/characters/pipoya/Dog 01-2.png"
-            },
-            {
-                "name": "dog3",
-                "img": "resources/characters/pipoya/Dog 01-3.png"
-            },
-            {
-                "name": "cat1",
-                "img": "resources/characters/pipoya/Cat 01-1.png"
-            },
-            {
-                "name": "cat2",
-                "img": "resources/characters/pipoya/Cat 01-2.png"
-            },
-            {
-                "name": "cat3",
-                "img": "resources/characters/pipoya/Cat 01-3.png"
-            }
+          {
+            "id": "dog1",
+            "name": "dog1",
+            "url": "resources/characters/pipoya/Dog 01-1.png",
+            "position": 0
```

Comment:
Do we need a position?
It seems ignored in the code. Furthermore, we are in an ordered list.